config/env_spec_request.go

```diff
@@ -71,12 +81,57 @@ type EnvironmentSpecRequest struct {
 	Request               *authv3.CheckRequest
 	authMan               auth.Manager
 	jwtResults            map[string]*jwtResult // JWTAuthentication.Name ->
-	verifier              jwt.Verifier
 	apiSpec               *APISpec
 	operation             *APIOperation
-	queryValues           url.Values
-	operationPath         string
 	consumerAuthorization *ConsumerAuthorization
+	variables             *requestVariables // for template reification
+}
+
+func (e *EnvironmentSpecRequest) parseRequest() {
+
+	path, queryString := func() (string, string) {
```

Comment:
What is the benefit of having an anonymous func here?

Reply:
Just encapsulation & self-documentation... avoiding some variable pollution. Do you think it would be clearer to remove it?

Reply:
No it's good as is ;-)